tests/assembly/ilog2.rs

```diff
@@ -0,0 +1,13 @@
+// assembly-output: bpf-linker
+// compile-flags: --crate-type cdylib -C link-arg=--unroll-loops -C link-arg=-O3
```

Comment:
do we need unroll-loops and 03? seems superfluous, see https://en.wikipedia.org/wiki/Chekhov%27s_gun

Reply:
Almost certainly copy-pasted this :grinning: